BCStudentSoftwareDevTeam/celts
# Refactor templates

## Issue Description

Fixes #1131 
- templates directory is messy and some files are in the wrong folder or inconsistent in format.

## Changes

- All events related files are now located in the events folder
- all files are formatted in camel case
- the outdated `addParticipants` html and js files have been deleted
- the outdated `outsideParticipants` model has been deleted

## Testing

- Run flask
- Run our test suite
- Check that Event List, Create Event, and other pages related to events and those under Admin still work.
